<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModsDialog</class>
 <widget class="QDialog" name="ModsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose a Modification</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="dialogGridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="custom_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="nameComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="colorLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sequence5Label">
       <property name="text">
        <string>sequence 5'</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sequence5LineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sequence3Label">
       <property name="text">
        <string>sequence 3'</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="sequence3LineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sequenceInternalLabel">
       <property name="text">
        <string>sequence internal</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="sequenceInternalLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="noteLabel">
       <property name="text">
        <string>Note</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="noteTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>custom_button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>custom_button_box</sender>
   <signal>rejected()</signal>
   <receiver>ModsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_button_box</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ModsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
